alembic/versions/a3d71270256c_adds_panel_application_fields.py

```diff
@@ -0,0 +1,79 @@
+"""Adds panel application fields
```

Comment:
I'm going to update the descriptions to specify the actual table names we're modifying. I can't figure out how to be more descriptive without actually just listing the columns we're adding ☹️